src/imitation/scripts/config/train_adversarial.py

```diff
@@ -178,7 +162,7 @@ def hopper():
 def humanoid():
   env_name = "Humanoid-v2"
   rollout_hint = "humanoid"
-  n_epochs = 2000
+  total_timesteps = 4e6
```

Comment:
My guess is Humanoid will need more timesteps than this. Certainly Humanoid will need more than 2x that of HalfCheetah, but maybe we're being very generous with HalfCheetah.